### ISSUE TITLE
描述错误

### ISSUE THREAD

**Issue by davidwang9527** [ShusenTang/Dive-into-DL-PyTorch#133]
**bug描述**
1.第一章"深度学习简介"中，“这样的思想后来被 PyTorch和MXNet的Gluon API 采用，后者也正是本书用来教学深度学习的工具。”，其中“后者”应该改为“前者”。
2.第二章"2.3.1"节中，建议将这句话去掉：“如果不想要被继续追踪，可以调用.detach()将其从追踪记录中分离出来，这样就可以防止将来的计算被追踪，这样梯度就传不过去了。” detach()会创建一个新的tensor，这新的tensor是从追踪记录中分离的，但是原tensor仍然会被追踪记录。对原tensor，要么直接修改.requires_grad为False，要么使用with torch.no_grad()包裹起来，使用detach()是无效的。
3.第二章"2.3.3"节中，"我么"改为"我们".
